.github/workflows/update-man-pages.yml

```diff
@@ -0,0 +1,39 @@
+name: Update man-pages workflow
+on:
+  schedule:
+    - cron: "42 3 1/15 * *"   # Trigger every 15 days at 03:42
+    #- cron: "0,5,10,15,20,25,30,35,40,45,50,55 * * * *"  # For testing
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-man-pages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository code
+        uses: actions/checkout@v4
+
+      - name: Update man-pages
+        run: |
+          set -euo pipefail
+
+          rm ./documentation/manpages/sdk/*
+
+          ./documentation/manpages/tool/run_docker.sh
+
+          if [[ -n $(git status -s) ]]; then
+            git config user.name 'TODO'
```

Comment:
I think it's very common to use the combination of:

```
git config user.name "github-actions"
git config user.email "github-actions@github.com"
```

e.g:

https://github.com/dotnet/msbuild/blob/1e7a0a86776c87955d195980fa1873447a3341af/.github/workflows/SyncAnalyzerTemplateMSBuildVersion.yml#L131-L134

Reply:
This makes sense to me!